src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -124,11 +127,20 @@ public void setPerson(Person target, Person editedPerson) {
     }
 
     @Override
-    public void updateFilteredPersonList(Predicate<Person> predicate) {
+    public void setModuleFilter(Predicate<Person> predicate) {
```

Comment:
I'd suggest changing the parameter to `modulePredicate` to be consistent with the usage of `searchPredicate` in the method below!

Reply:
Done!